src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -39,9 +42,15 @@ private static void setupDrivers(Application application) {
 		DriverFeature.addDriver("Logger Driver", loggerDriver);
 		DriverFeature.getDriverManager().setCurrentDriver(loggerDriver);
 
-		Job2dDriver testDriver = new MyAdapter();
+		Job2dDriver testDriver = new DrawLinesByMoving();
 		DriverFeature.addDriver("Buggy Simulator", testDriver);
 
+        Job2dDriver dottedDriver = new LineDrawerAdapter(LineFactory.getDottedLine());
+		DriverFeature.addDriver("Dotted Simulator", dottedDriver);
+
+        Job2dDriver specialDriver = new LineDrawerAdapter(LineFactory.getSpecialLine());
+		DriverFeature.addDriver("Special Simulator", specialDriver);
+
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.